src/Modules/ConfigOptions.lua

```diff
@@ -1209,6 +1209,9 @@ Huge sets the radius to 11.
 	{ var = "conditionHaveManaStorm", type = "check", label = "Do you have Manastorm's ^xADAA47Lightning ^7Buff?", ifFlag = "Condition:HaveManaStorm", tooltip = "This option enables Manastorm's ^xADAA47Lightning ^7Damage Buff.\n(When you cast a Spell, Sacrifice all ^x7070FFMana ^7to gain Added Maximum ^xADAA47Lightning ^7Damage\nequal to 25% of Sacrificed ^x7070FFMana ^7for 4 seconds)", apply = function(val, modList, enemyModList)
 		modList:NewMod("Condition:SacrificeManaForLightning", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
 	end },
+	{ var = "conditionHaveVixensEntrapment", type = "check", label = "Use Vixen's Entrapment cooldown?", ifFlag = "Condition:HaveVixensEntrapment", tooltip = "Causes Impending Doom calculations to take the cooldown of Vixen's Entrapment into account,\nlowering DPS if cast rate is higher than the gloves' trigger cooldown.", apply = function(val, modList, enemyModList)
+		modList:NewMod("VixenTriggerCap", "FLAG", true, "Config", { type = "Condition", var = "Combat" })
+	end },
```

Comment:
This should probably always apply.

Reply:
There are several ways to trigger Doom Blast. One example is putting an Anomalous Impending Doom into Dialla's Malefaction to get 100% reduced curse duration. I can see such a player still wanting to use Vixen's Entrapment for the easy extra curse application. In this case, they are not limited by the glove cooldown even if they are both wearing the glove and using Impending Doom. That is why it needs to be an option.

However, it should probably be enabled by default when you equip the glove. If I figure out how to do that. :D

Reply:
The spectre curses are set to be default on so just use that code.